core/commands/storage.go

```diff
@@ -532,17 +586,17 @@ func retryProcess(ctx context.Context, n *core.IpfsNode, api coreiface.CoreAPI,
 	}
 	// send over contract
 	c := new(guardpb.Contract)
-	err = proto.Unmarshal(halfSignedGuardContract, c)
+	err = proto.Unmarshal(shard.HalfSignedGuardContract, c)
```

Comment:
I don't think `c` is used anywhere, can you remove this line because `shard.HalfSignedGuardContract` is already set?